docs/src/regression-testing.md

```diff
@@ -56,7 +56,16 @@ In particular, the Kani testing suites are composed of:
                 Arguments can be specified in the `Cargo.toml` configuration file.
                 Similar to the `expected` suite, we look for `*.expected` files
                 for each harness in the package.
-* `cargo-ui`: Similar to `cargo-kani`, but focuses on the user interface like the `ui` test suite.
+ * `cargo-ui`: Similar to `cargo-kani`, but focuses on the user interface like the `ui` test suite.
+ * `exec-init`: This suite is useful to execute script-based tests, and also
+                allows checking expected output and exit codes after running them. The suite
+                expects a folder with a `config.yml` file within, which should contain:
```

Comment:
We should clarify the restriction on the directory structure, e.g. "Tests are expected to be placed directly under `exec-init` in a directory with a `config.yml`.".

Reply:
Done (see new subsection mentioned in `Script-based tests`).